metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_v2.py

```diff
@@ -1689,7 +1676,8 @@ def close(self) -> None:
         super().close()
         StatefulIngestionSourceBase.close(self)
         self.view_definitions.close()
-        self.sql_parser_schema_resolver.close()
+        # Do we not need to close filebackeddicts opened explicitly ?
```

Comment:
it's nicer to close them explicitly, but they'll get automatically closed when the objects get garbage collected